changelogs/unreleased.md

```diff
@@ -0,0 +1,40 @@
+# Changelog Template
+
+The format is based on [Rust's Release Notes](https://github.com/rust-lang/rust/blob/master/RELEASES.md),
+with the exception of splitting the changelog into multiple files for readability purposes.
+
+## [Unreleased]
+
+### Language
+
+- Aliases
```

Comment:
Should we link to the PR as Rust does?

Reply:
We could I'd have to find a lot of them since they are older.